lib/request.js

```diff
@@ -149,6 +149,10 @@ Object.defineProperties(Request.prototype, {
       return this.raw.url
     }
   },
+  originalUrl: {
+    value: '/',
```

Comment:
why is it a slash by default?

Reply:
If I set it to `this.url`, then it fails the request test
```typescript
​ FAIL ​ test/internals/request.test.js
 ✖ should be equal

  test/internals/request.test.js
  61 |   t.equal(request.method, 'GET')
  62 |   t.equal(request.url, '/')
> 63 |   t.equal(request.originalUrl, '/')
     | ----^
  64 |   t.equal(request.socket, req.socket)
  65 |   t.equal(request.protocol, 'http')
  66 |   t.equal(request.routerPath, context.config.url)

  test: test/internals/request.test.js Regular request
  found: null
  wanted: /
  compare: ===
  stack: |
    Test.<anonymous> (test/internals/request.test.js:63:5)
    Object.<anonymous> (test/internals/request.test.js:15:1)
    Module.replacementCompile (node_modules/append-transform/index.js:60:13)
    Object.<anonymous> (node_modules/append-transform/index.js:64:4)
```

I can't have a getter because then getters cannot be set dynamically.

Reply:
Which one should I use?

Reply:
@mcollina ping

Reply:
I think it should default to the `this.raw.url` by default. See my comment below about the `rewriteUrl` 👍

Reply:
You mean, remove [Line 472](https://github.com/fastify/fastify/blob/a19b93c143d862fc645d2ccadac7ad98a569494f/lib/route.js#L472) and there, use
```js
 originalUrl: {
    get () {
      if (!this[kRequestOriginalUrl]) {
        this[kRequestOriginalUrl] = this.raw.originalUrl
      }
      return this[kRequestOriginalUrl]
    }
  },
```
?


I tested this and it fails the test. When I logged it, it says, `this.raw.originalUrl` is null